backend/open_webui/retrieval/utils.py

```
@@ -247,26 +262,31 @@ def get_embedding_function(
-        return lambda query: embedding_function.encode(query).tolist()
+        return lambda query, user=None: embedding_function.encode(query).tolist()
     elif embedding_engine in ["ollama", "openai"]:
-        func = lambda query: generate_embeddings(
+        func = lambda query, user=None: generate_embeddings(
             engine=embedding_engine,
             model=embedding_model,
             text=query,
             url=url,
             key=key,
+            user=user,
         )
```

Ruff (0.8.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `func` as a `def`

(E731)